backend/src/Designer/Controllers/ResourceAdminController.cs

```diff
@@ -45,5 +46,29 @@ public ActionResult<List<ServiceResource>> GetRepositoryResourceList(string org)
             List<ServiceResource> repositoryResourceList = _repository.GetServiceResources(org, string.Format("{0}-resources", org));
             return repositoryResourceList != null && repositoryResourceList.Count > 0 ? repositoryResourceList : StatusCode(204);
         }
+
+        [HttpGet]
+        [Route("designer/api/{org}/resources/repository/{id}")]
```

Comment:
Will it work with two routes
[Route("designer/api/{org}/{repo}/resource/repository/{id}")]

hvor repo either is an app repo where id is not required or a resource repo where id is required?

Reply:
My idea here was that the "id" in the route represents either the name of an app like this: "repository:<reponame>" or an identifier of a resource like this: "id:<resourceId>". But you are thinking about adding a second route to the endpoint?